### DISCUSSION TITLE
Welcome to qiskit-textbook Discussions!

### DISCUSSION THREAD

**Discussion by frankharkins** [qiskit-community/qiskit-textbook#858] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. This is the place to:
  * Ask any questions about the Qiskit Textbook and its content.
  * Share and develop ideas.
  * Engage with other community members.

 To get started, comment below with an introduction of yourself and how you use the textbook.

**Comment by danijmarko** [comment]
Hello, 
I believe there is a problem with the loss function used in Hybrid quantum classical networks with qiskit.
You use the NLLLoss function which by definition takes the logarithm of probability as argument (https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html), but instead you give it directly the probability as argument. I believe this is why your loss function takes negative values rather then positive ones as it is supposed to.

Yet somehow, the network learns with such ill-defined function and not when I replace probability by the logarithm of probability.
Do you have an explanation for it?

**Comment by physicsshubham** [comment]
hello everyone.
I was wondering why don't we add a section, where can give an explanation of the superposition in quantum mechanics. and how qubits utilize these superposition in quantum computation. and some examples of physical pictures of qubits.
